canonical/lxd-ui
# Add powerflex driver support

## Done

- Add powerflex driver support for storage pools

Fixes #677

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - to test this we need a lab access, as it is not possible to run powerflex locally or in the demo environment. You can explore the options though on "Create new storage pool" after selecting the "Dell powerflex" driver.
    - see [docs on powerflex](https://documentation.ubuntu.com/lxd/en/latest/reference/storage_powerflex/#storage-pool-configuration)
    - see [spec on powerflex](https://discourse.ubuntu.com/t/dell-powerflex-storage-driver/40611)
